content/correspondence/reference/API endpoints/initialize/_index.en.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Initialize
+linktitle: Initialize
+description: Endpoint for initializing a correspondence.
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence
+
+## Description
+This endpoint prepares and queues a correspondence for sending. Before using the endpoint and attachment must be uploaded by using the following endpoint in order to populate the "ExistingAttachments" field:
```

Comment:
Typo: Before using the endpoint **an** attachment must be uploaded

Reply:
It should be Attachments, not singular, 

Also, should probably state that existingAttachments is optional. For example: 

This endpoint prepares and queues a correspondence for sending. Before using this endpoint any correlating attachments must be uploaded beforehand to populate the "ExistingAttachments" field.